ODIN_II/SRC/netlist_create_from_ast.cpp

```diff
@@ -204,8 +204,13 @@ void look_for_clocks(netlist_t* netlist) {
 
     for (i = 0; i < netlist->num_ff_nodes; i++) {
         oassert(netlist->ff_nodes[i]->input_pins[1]->net->num_driver_pins == 1);
```

Comment:
I think this needs some commenting. What is this trying to do and why is it needed?

Reply:
@sdamghan